ar/index.html

```diff
@@ -31,65 +31,65 @@
 </head>
 <body dir="rtl">
   <main>
-    <h1>!لا تسأل لتسأل، فقط إسأل </h1>
+    <h1>لا تسأل لتسأل، فقط اسأل</h1>
     <p>
-  بين الحين والأخر يأتي احدهم في غرف الدردشة قائلا شيئ مشابه لـ  :  </p>
+      بين الحين والآخر يأتي أحدهم في غرف الدردشة قائلاً شيء مشابه لـ  :
+    </p>
     <blockquote>
       <span class="name">فلان:</span>
       <p class="message">
         هل هناك محترفين جافا هنا؟
       </p>
     </blockquote>
     <p>
-        هذه الطريقة سيئة لأسباب عدة. ما يقصده الشخص
+      هذه الطريقة سيئة، لأسباب عدة. ما يقصده الشخص
       <em>في الحقيقة </em>هو 
     </p>
     <blockquote>
       <span class="name">فلان:</span>
       <p class="message">
- هل هناك اي محترف جافا يقبل مساعدتي في حل المشكلة، مهما كانت، حتى ولو لم تكن لها علاقة ب"جافا" او ان كان هناك اي شخص لا عرف شيئ عن الجافا يستطيع حل مشكلتي؟     </p>
+        هل هناك أي محترف جافا يقبل مساعدتي في حل المشكلة، مهما كانت، حتى ولو لم تكن لها علاقة بجافا أو إن كان هناك أي شخص لايعرف شيئاً عن الجافا يستطيع حل مشكلتي؟
+      </p>
     </blockquote>
     <p>
-  هناك عدة اسباب تجعل الذين لهم خبرة لا يجيبوا عن سؤالك. سؤالك يسأل أكثر مما تظن.
+      هنالك عدة أسباب تجعل الذين يمتلكون الخبرة لايجيبوا عن سؤالك. سؤالك يسأل أكثر مما تظن.
     </p>
     <p>
-  انت تسأل الناس لتحملهم المسؤالية. انت تختبر ثقة الناس في قدراتهم. أنت أيضًا تمنع أشخاصًا آخرين من الإجابة بلا ضرورة.
-
+      أنت تسأل الناس لتحملهم المسؤولية. أنت تختبر ثقة الناس في قدراتهم. أنت أيضاً تمنع أشخاصاً آخرين من الإجابة بلا ضرورة.
     </p>
     <p>
-  يمكن ايضا رؤية سؤالك بهذه الطريقة...
+      بدلاً من ذلك، يمكن رؤية سؤالك على أنه...
     </p>
     <blockquote>
       <span class="name">فلان:</span>
       <p class="message">
-   لدي سؤال عن  الجافا لكني اشعر بالكسل لأقوم بصياغته على شكل كلمات الا لو كان بالقناة شخص شخص يستطيع حله.
-
+        لدي سؤال حول الجافا لكني أشعر بالكسل لأقوم بصياغته على شكل كلمات إلا إذا كان بالقناة شخص قد يكون قادراَ على الإجابة عنه.
       </p>
     </blockquote>
     <p>
-    هو مجرد كسل. إن لم تكن راغبا بالقيام بأي عمل لمساعدة نفسك. لماذا نقوم به نحن؟... 
+      ... إنه مجرد كسل. إن لم تكن راغباً بالقيام بأي عمل لحل مشكلتك، فلماذا يجب علينا؟
```

Comment:
why?

Reply:
"إنه" - Better meaning because of small sentence
"راغباً" - Nunation
"لحل مشكلتك، فلماذا يجب علينا؟":
- The old one: ... anything to help yourself. why do we do it?
- The new one: ... anything to solve your problem, why should we?

Changed the three dots "..." position to be at first, you will see them at last but it will be rendered first because of Arabic blending inside codes.

Reply:
I think "مساعدة نفسك" has a better meaning than "لحل مشكلتك" 
The rest is good